kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -860,7 +860,7 @@ namespace Testing {
 
         // Check edge distances
         const auto &r_elem_dist_edge = (volume_part.ElementsBegin())->GetValue(ELEMENTAL_EDGE_DISTANCES);
-        const std::vector<double> expected_values_edge = {-1,-1,-1,0.961531,-1,-1};
+        const std::vector<double> expected_values_edge = {-1,-1,-1,0.959824,-1,-1};
```

Comment:
With the new tolerance, this `ELEMENTAL_EDGE_DISTANCES` are computed with a slightly different `avg_pt` in:

https://github.com/KratosMultiphysics/Kratos/blob/c7bc97cfd7bcbfe1b03d29b187323543ea8644c7/kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp#L346

This avg_pt was computed with the following point with the previous tolerance:

`aux_pts : [[3](0.68381,0.797056,0.479135)]`

And with the new tolerance with the following two points (that were considered the same point before).

`aux_pts : [[3](0.68381,0.797056,0.479135), [3](0.68375,0.797069,0.478814)]`

In this case the effect is just a slight change in the distance value.